WcaOnRails/spec/features/sign_up_spec.rb

```diff
@@ -82,6 +83,75 @@
 
       expect(page).to have_selector('#have-competed', visible: true)
     end
+
+    it "remembers their selected wca id on validation error" do
+      visit "/users/sign_up"
+      click_on "I have competed in a WCA competition."
+      # They have not selected a valid WCA ID yet, so don't show the birthdate verification
+      # field.
+      expect(page.find("div.user_dob_verification", visible: false).visible?).to eq false
```

Comment:
Good point! I've cleaned up all the occurences of this in the file I could find.